ui/src/routes/_layout/admin/index.tsx

```diff
@@ -241,31 +270,120 @@ const OverviewContent = () => {
+          <CardHeader>
+            <CardTitle className='text-sm'>Create User</CardTitle>
+            <CardDescription>Create a new user to ORT Server.</CardDescription>
           </CardHeader>
-          <CardContent>
-            <div className='gap-1'>
-              Manage users in{' '}
-              <a
-                href={
-                  authBaseUrl + '/admin/master/console/#/' + realm + '/users'
-                }
-                target='_blank'
-                className='gap-1 text-blue-400 hover:underline'
-              >
-                <span>Keycloak</span>
-                <ExternalLink className='mb-1 ml-1 inline' size={16} />
-              </a>
-            </div>
-          </CardContent>
+          <Form {...form}>
+            <form onSubmit={form.handleSubmit(onSubmit)}>
+              <CardContent className='space-y-4'>
+                <FormField
+                  control={form.control}
+                  name='username'
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Username</FormLabel>
+                      <FormControl>
+                        <Input {...field} autoFocus />
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name='password'
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Password</FormLabel>
+                      <FormControl>
+                        <Input
+                          type='password'
+                          {...field}
+                          placeholder='(optional)'
+                        />
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name='temporary'
+                  render={({ field }) => (
+                    <FormItem className='flex flex-row items-start space-x-3 space-y-0 rounded-md border p-4'>
+                      <FormControl>
+                        <Checkbox
+                          checked={field.value}
+                          onCheckedChange={field.onChange}
+                        />
+                      </FormControl>
+                      <div className='space-y-1 leading-none'>
+                        <FormLabel>Password is temporary</FormLabel>
+                        <FormDescription>
+                          When a temporary password is assigned, the user has to
+                          change it on first login. If you are creating a user
+                          for demonstration purposes, it is easier to leave this
+                          unchecked.
+                        </FormDescription>
```

Comment:
I wonder whether having both the label and description, we should have only a more descriptive label, like "User has to change the password on first login" or "Password change required on first login".

Reply:
I like this as well.